python/rtypes/dataframe/trigger_manager.py

```diff
@@ -0,0 +1,230 @@
+from rtypes.pcc.triggers import TriggerAction, TriggerTime
```

Comment:
Put this after the normal imports.
sequence of imports should always be:

imports from built in modules
imports from local modules.